deploy/example_config/eks-gp2-for-kd.yaml

```diff
@@ -0,0 +1,31 @@
+# This is an example of one solution for the default StorageClass on EKS
+# not setting its volumeBindingMode to WaitForFirstConsumer. It creates
+# another storage class, which is identical to the "gp2" class except with
+# the necessary setting for volumeBindingMode, and then configures KD to use
+# that storage class (even though it is not the K8s default storage class).
+
+# The kubedirector-support label ensures that this storage class will be
+# removed by "make teardown". This is purely a behavior of the Makefile in
+# this repo and is not a fundamental KubeDirector functionality.
+
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: gp2-for-kd
+  labels:
+    kubedirector-support: "true"
+parameters:
+  fsType: ext4
+  type: gp2
```

Comment:
It looks like both of these are the default value - do we need to include them? Can we leave the `parameters` block empty (or delete it entirely)?

Reply:
Seems to still work when omitted, but no default values are filled in for the created resource, so it is a mystery what sort of PV it will create until it has been created. I think I'd rather leave it explicit.

Reply:
Fair.